src/main/java/jetbrains/buildServer/iaa/common/Constants.java

```diff
@@ -24,9 +24,16 @@
   // Parameter names
   public static final String DEFAULT_RESPONSIBLE = "teamcity.iaa.defaultResponsible";
 
+  // Parameter names
+  public static final String SILENT_MODE_ON = "teamcity.iaa.silentModeOn";
+
   //Message constants
-  public static final String REASON_PREFIX = "Auto-assigned investigation:";
+  public static final String REASON_PREFIX_ASSIGN = "You were chosen as probable responsible because you";
```

Comment:
Typo - "You was ..."

Reply:
https://www.quora.com/Grammatically-which-is-correct-%E2%80%9Cyou-was%E2%80%9D-or-%E2%80%9Cyou-were%E2%80%9D-where-%E2%80%9Cyou%E2%80%9D-is-singular